lib/ansible/modules/windows/win_certificate_store.py

```diff
@@ -118,6 +118,10 @@
   Kerberos with credential delegation, or use C(become) to bypass these
   restrictions.
 - The certificates must be located on the Windows host to be set with I(path).
+- When importing a certificate for usage in IIS, it is generally required
+  to use the C(machine) key_storage option, as both C(default) and C(user)
+  will make the private key unreadable to IIS APPPOOL identities and prevent
```

Comment:
It's less the private key but the whole certificate and key not being accessible to that account.

Reply:
Not sure this is correct unless you use the `CurrentUser` certificate store. `LocalMachine` is the default.

Reply:
> LocalMachine is the default

I disagree, we have seen how the default option on Windows is not actually LocalMachine, this is the whole point of this PR to point out how the default may not be what you need :)

> I was able to view and select the certificate from a different user account without issue

I'm not able to do so, when I go to select a certificate I am only able to see the certs in the Personal store of LocalMachine, any certs in the Personal store are not viewable and therefore not selectable.

Reply:
I went ahead and added `store_location: LocalMachine` to the example, I agree that it is probably a good idea not to have the documentation rely on the default setting to be correct for the use case as the default could change.

Reply:
Ahh sorry, complete mind blank there and totally forgot we are talking about `key_storage` and not `store_location`. Please ignore this comment.